jobs/release.Jenkinsfile

```diff
@@ -96,11 +99,16 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             """)
         }
 
-        def basearches = ""
-        if (params.ARCHES == '') {
-            basearches = shwrapCapture("jq -r '.builds | map(select(.id == \"${params.VERSION}\"))[].arches[]' builds/builds.json").split()
-        } else {
-            basearches = params.ARCHES.split()
+        def basearches = params.ARCHES.split()
+        def builtarches = shwrapCapture("jq -r '.builds | map(select(.id == \"${params.VERSION}\"))[].arches[]' builds/builds.json").split()
+        if builtarches.toSet() != basearches.toSet() {
+            if params.ALLOW_MISSING_ARCHES {
+                echo "Some requested architectures did not successfully build! Continuing."
+                basearches = builtarches
+            else {
+                echo "ERROR: Some requested architectures did not successfully build"
```

Comment:
Could we print here `basearches` and `builtarches` to make it easier to debug?

Reply:
will do